specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -1040,6 +1178,21 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-access-soft-deleted-resources",
```

Comment:
With this parameter , does it mean only soft deleted items are returned, or all objects include soft deleted ones are returned?
If a VG is soft deleted, will the volumes in the VG be list out with this parameter?
Please update description to include this.